### ISSUE TITLE
Question: Set Prettier to default

### ISSUE THREAD

**Issue by ginoarpesella** [prettier/prettier-vscode#477]
I love the restyling of my files but can I change my vscodes default indentations and line spacing... etc, to that of Prettier. Currently my code looks rubbish while I'm coding because of the different indentations shared. How and I uniform my experience. Thanks

**Comment by mubaidr** [comment]
`Prettier` is set as default Formatter for all supported file types when installed, unless you have some other Formatter installed which overrides prettier. As mentioned in Readme:

> A word of warning-if you have any other code formatting extensions installed such as for example hugely popular HookyQR.beautify or taichi.react-beautify they might take precedence and format your code instead of Prettier leading to unexpected results.

**Comment by mubaidr** [comment]
Sorry, let me try again: 

Create an `.editorconfig` file in your project root with
```js
indent_style = space
indent_size = 2
```
Now both the editor and prettier will follow same rules of indentation for each file in your project. Details: https://editorconfig.org/

You can also set the `tabWidth` and `insertSpaces` in your editor settings. This way editor will follow the same setting untill prettier take over. 

Also this `"editor.formatOnType": true,` could be you need. It will run formatter (prettier in you case) as you type to update correct indentation.

**Comment by CiGit** [comment]
We do not register prettier for formatOnType.

This could be an idea tho. 
When prettier runs on Document **D**, update `tabWidth` and `insertSpace` for **D**. Based on settings we read. (This doesn't work with eslint ...)
